beginner_source/basics/intro.py

```diff
@@ -13,9 +13,9 @@
 ===================
 
 Authors:
-`Suraj Subramanian <https://github.com/suraj813>`_,
+`Suraj Subramanian <https://github.com/subramen>`_,
 `Seth Juarez <https://github.com/sethjuarez/>`_,
-`Cassie Breviu <https://github.com/cassieview/>`_,
+`Cassie Breviu <https://github.com/cassiebreviu/>`_,
```

Comment:
@cassiebreviu - can you confirm this is the new account that should be listed here.

Reply:
Yes `https://github.com/cassiebreviu/` is correct. Thanks!